plugins/volume/org.xen.xapi.storage.zfs-ng/zfsutils.py

```diff
@@ -0,0 +1,38 @@
+import os
+
+from xapi.storage.common import call
+
+MOUNT_ROOT = '/var/run/sr-mount'
+
+def pool_mountpoint(dbg, pool_name):
+    cmd = "zfs get mountpoint -H -o value".split() + [ pool_name ]
+    return call(dbg, cmd).strip()
+
+def pool_create(dbg, pool_name, devs):
+    cmd = [
```

Comment:
nit and it is ok for me but why not splitting cmd like others?

Reply:
I pondered that, but I felt it as more clear to keep the options together with their arguments, which only left "zpool create" to split, so that would look quite weird.  But then I left the `.split()` for `destroy` so yes it is in fact lacking consistency